Build Error

Error is as follows:
```
--- stderr: controller_manager                                           
/home/jayalekshmijayakumar/.local/lib/python3.10/site-packages/setuptools/_distutils/cmd.py:66: SetuptoolsDeprecationWarning: setup.py install is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` directly.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
        ********************************************************************************

!!
  self.initialize_options()
during RTL pass: ud_dce
In file included from /opt/ros/humble/include/rclcpp/rclcpp/any_executable.hpp:25,
                 from /opt/ros/humble/include/rclcpp/rclcpp/memory_strategy.hpp:25,
                 from /opt/ros/humble/include/rclcpp/rclcpp/memory_strategies.hpp:18,
                 from /opt/ros/humble/include/rclcpp/rclcpp/executor_options.hpp:20,
                 from /opt/ros/humble/include/rclcpp/rclcpp/executor.hpp:37,
                 from /opt/ros/humble/include/rclcpp/rclcpp/executors/multi_threaded_executor.hpp:25,
                 from /opt/ros/humble/include/rclcpp/rclcpp/executors.hpp:21,
                 from /opt/ros/humble/include/rclcpp/rclcpp/rclcpp.hpp:155,
                 from /home/jayalekshmijayakumar/workspaces/ur_gazebo/install/controller_interface/include/controller_interface/controller_interface_base.hpp:28,
                 from /home/jayalekshmijayakumar/workspaces/ur_gazebo/install/controller_interface/include/controller_interface/chainable_controller_interface.hpp:20,
                 from /home/jayalekshmijayakumar/workspaces/ur_gazebo/src/ros2_control/controller_manager/include/controller_manager/controller_manager.hpp:26,
                 from /home/jayalekshmijayakumar/workspaces/ur_gazebo/src/ros2_control/controller_manager/src/controller_manager.cpp:15:
/opt/ros/humble/include/rclcpp/rclcpp/subscription.hpp: In member function ‘void rclcpp::Subscription<MessageT, AllocatorT, SubscribedT, ROSMessageT, MessageMemoryStrategyT>::return_message(std::shared_ptr<void>&) [with MessageT = std_msgs::msg::String_<std::allocator<void> >; AllocatorT = std::allocator<void>; SubscribedT = std_msgs::msg::String_<std::allocator<void> >; ROSMessageT = std_msgs::msg::String_<std::allocator<void> >; MessageMemoryStrategyT = rclcpp::message_memory_strategy::MessageMemoryStrategy<std_msgs::msg::String_<std::allocator<void> >, std::allocator<void> >]’:
/opt/ros/humble/include/rclcpp/rclcpp/subscription.hpp:402:3: **internal compiler error:** Segmentation fault
  402 |   }
      |   ^
0x7491b064251f ???
	./signal/../sysdeps/unix/sysv/linux/x86_64/libc_sigaction.c:0
Please submit a full bug report,
with preprocessed source if appropriate.
Please include the complete backtrace with any bug report.
See <file:///usr/share/doc/gcc-11/README.Bugs> for instructions.
gmake[2]: *** [CMakeFiles/controller_manager.dir/build.make:76: CMakeFiles/controller_manager.dir/src/controller_manager.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:208: CMakeFiles/controller_manager.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
---
Failed   <<< controller_manager [14.5s, exited with code 2]
Aborted  <<< moveit_msgs [25.9s]                                    

Summary: 14 packages finished [26.1s]
  1 package failed: controller_manager
  1 package aborted: moveit_msgs
  5 packages had stderr output: controller_manager moveit_configs_utils ros2_controllers_test_nodes rqt_controller_manager rqt_joint_trajectory_controller
  58 packages not processed
```  
  I have been getting the following error when building the package using the following command:
  colcon build --symlink-install --mixin rel-with-deb-info compile-commands ccache

--- a/README.md
+++ b/README.md
@@ -16,8 +16,8 @@ Ubuntu 22.04 at the time of writing. However, it is no longer supported.
   </tr>
   <tr>
     <th>Branch</th>
-    <td><a href="https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/tree/humble">humble</a></td>
-    <td><a href="https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/tree/iron">iron</a></td>
+    <td><a href="https://github.com/UniversalRobots/Universal_Robots_ROS2_Gazebo_Simulation/tree/humble">humble</a></td>
+    <td><a href="https://github.com/UniversalRobots/Universal_Robots_ROS2_Gazebo_Simulation/tree/iron">iron</a></td>
   </tr>
   <tr>
     <th>Build status</th>
@@ -58,18 +58,16 @@ Skip any of below steps is not applicable.
 
 1. Download the required repositories and install package dependencies:
    ```
-   cd $COLCON_WS
-   git clone git@github.com:UniversalRobots/Universal_Robots_ROS2_Gazebo_Simulation.git src/Universal_Robots_ROS2_Gazebo_Simulation
-   vcs import src --input src/Universal_Robots_ROS2_Gazebo_Simulation/Universal_Robots_ROS2_Gazebo_Simulation.<ros-distro>.repos
-   rosdep install --ignore-src --from-paths src -y
-   cd ..
+   cd $COLCON_WS/src
+   git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Gazebo_Simulation.git
+   rosdep update && rosdep install --ignore-src --from-paths . -y
    ```
 
 ### Configure and Build Workspace:
 To configure and build workspace execute following commands:
   ```
   cd $COLCON_WS
-  colcon build --symlink-install --mixin rel-with-deb-info compile-commands ccache
+  colcon build --symlink-install
   ```
 
 ## Running Simulation

--- a/Universal_Robots_ROS2_Gazebo_Simulation.iron.repos
+++ b/Universal_Robots_ROS2_Gazebo_Simulation.iron.repos
@@ -7,7 +7,7 @@ repositories:
   gazebo_ros2_control:
     type: git
     url: https://github.com/ros-simulation/gazebo_ros2_control.git
-    version: master
+    version: iron
 
   moveit2:
     type: git

